### ISSUE TITLE
[FR] folder view in collections module should display total number of images recursive

### ISSUE THREAD

**Issue by Solarer** [darktable-org/darktable#11799]
**Is your feature request related to a problem? Please describe.**
Looking at the folder view in the collections module it is not clear to me how many images are located in a folder (or whether the folder is completely empty). Darktable only displays the number of images in a folder but does not take into account images in sub folders. 

The number of images in the top level of a folder mean nothing to me when I have a folder structure like this: year | month | day -> pictures. In this case the year and month folders do not have a number and appear to be empty. Only the day folders have a number. And if I copy 10 pictures by accident into the year folder it looks like I shot only 10 pictures in that whole year.

**Describe the solution you'd like**
The total number of images in a folder and all of its subfolders should be displayed.

**Additional context**
![image](https://user-images.githubusercontent.com/2805109/168393648-1e7e0785-8759-453d-be8d-bfb5c9325928.png)

**Comment by difrkaguilar** [comment]
This feature is already present in the developers version.
![Screenshot-20220514100003-351x528](https://user-images.githubusercontent.com/67672868/168429898-0fe44419-e2c4-48a1-bb50-6b3469cd00a5.png)

**Comment by Solarer** [comment]
No, this is still the case in the latest dev version. You are looking for collections by "capture date". If I chose that setting darktable will correctly show me how many images I took in 2022 and so on. But I am using the "folder" view - I simply have a folder called "2022". In folder view, the number of images is based on the number of images in the folder itself. Child folder content is not taken into account

Folder view (not correct)
![image](https://user-images.githubusercontent.com/2805109/181795235-e25984b1-f3db-4923-8580-aca5f6bb8717.png)

Capture date (correct)
![image](https://user-images.githubusercontent.com/2805109/181795334-e336a473-e0ef-477c-9428-b1910fba7ae7.png)
